src/animation_curses.cpp

```diff
@@ -1,4 +1,4 @@
-#if (!defined SDLTILES)
```

Comment:
It's not a canonical style if it's not declared to be canonical.  The "why not" is because it creates noise.